moztelemetry/spark.py

```diff
@@ -182,16 +182,20 @@ def _get_ping_properties(ping, paths, only_median, with_processes,
                                            .get("content", {}) \
```

Comment:
Mauro and I were discussing this and think that using a safe dict as the cursor might be a better (and more readable) solution. See [What is the best way to implement nested dictionaries in Python?](http://stackoverflow.com/questions/635483/what-is-the-best-way-to-implement-nested-dictionaries/19829714#19829714)

Then we can replace all `.get(elem)` with just `[elem]`